### DISCUSSION TITLE
Some projects I've made

### DISCUSSION THREAD

**Discussion by Paraxdev** [mantinedev/mantine#351] [Show]
![image](https://user-images.githubusercontent.com/29353794/139333015-54149615-9221-4a34-8e7b-2d16a0e00a98.png)
 (https://phentix.vercel.app/ )
 
 
![image](https://user-images.githubusercontent.com/29353794/139333071-dfc9f8c9-27db-4d3c-a2f5-e88884c7667a.png)
( https://excellent-dynasty.vercel.app/ )

**Comment by Paraxdev** [comment]
Note on excellent-dynasty: Animated BG and header from mantine docs

**Comment by rtivital** [comment]
Thanks for sharing!
